alpine/Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM alpine:3.3
 
+RUN apk --update add \
+	ca-certificates \
+	&& rm -rf /var/cache/apk/*
```

Comment:
You should use `apk add --no-cache ca-certificates` here and then we don't need either `--update` or the `rm` :+1: